log/ui.go

```diff
@@ -14,6 +39,106 @@ func lvlUI(l int, args ...interface{}) {
 	}
 }
 
+// New creates a new Logger
+func New(out io.Writer, prefix string, flag int) *LoggerPanic {
+	return &LoggerPanic{out: out, prefix: prefix, flag: flag}
```

Comment:
What is all of this about?

This task is about making it so that when you call something like log.Panic("The number is %d", 3) you get a panic message of "The number is 3", and not a useless message like you get today.

We do not need/want a new kind of logger, we want the current log.Panic to format the string.